### PR TITLE
Remove SAML IDP feature

### PR DESCRIPTION
**Pull request by hsinn0**
- In preparation for the SAML library replacement

~~Approve or disapprove but do not merge to develop yet.~~ We want merge this to `develop` now. As this is a breaking change, ~~we have to decide which UAA version this change should go with.~~ we also want to bump UAA major version after the merge.

### CODE REVIEW

**Review by strehle — COMMENTED** [review]
Check and change Readme notice but the rest LGTM

**Comment by strehle** [comment]
As mentioned in slack , please wait until release 76.31.0, then merge this PR and increase major version, thanks

File: README.md [hunk 1]
```diff
@@ -339,3 +337,8 @@ To debug UAA and LDAP integrations, we use an OpenLdap docker image from [VMWare
 
 Use below command to clean-up container and volume:
 - `docker-compose down --volumes`
+
+# UAA's functionality as a SAML identity provider (IdP)
```
Comment (swalchemist) [inline README.md]: I don't think we need to document in the README a feature that is no longer present. That's what the release notes are for. Removing the deprecation notice is a sufficient change.

**Review by swalchemist — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (hsinn0) [reply README.md]: Removed the added section.

**Comment by hsinn0** [comment]
> Check and change Readme notice but the rest LGTM

Done

**Review by swalchemist — APPROVED** [review]

**Review by strehle — APPROVED** [review]

**Pull request merged**